MERN/Rendering_Element .md

```diff
@@ -0,0 +1,69 @@
+## Rendering Elements
+
+<p> Elements are the Smallest building blocks of React app.
+  It describes what you want to see on the screen.
+  Example of an element is :-<p>
+
+```
+
+const ele = <h1> Rendering Elements in react </h1>;
+
+```
+
+<p>React DOM update the DOM with the content in the react element. React elements are cheap to create, and are plain objects. Its a simple object that describes DOM node and its attributes.
+It is an immutable (which can't be changed) object and methods can't be apply on them.</p>
+
+## Rendering an Element into DOM
+
+<p>For rendering any element into browser DOM, we should have a container or root element.
+  So, We will have parent div somewhere in our HTML file with id="root" or id="app".</p>
+
+```
+
+<div id="root"></div>
+
+```
+<p> ReactDOM manages everything that is inside this div and this root is called as DOM node </p>
+
+<br>
+
+<p> Now, for rendering simple React element to root node, one has to write following code in App.js file </p>
```

Comment:
Correction: It is not mandatory to name it App.js, It is just a convention which everyone generally follow. Everything should be inside the file which is taken as a first parameter when using React.render method.